test/specs/carousel.test.js

```diff
@@ -261,6 +261,20 @@ describe('<Carousel />', () => {
   });
 
   describe('methods', () => {
+    it("should call onResize prop on onResize handler", () => {
+      const wrapper = mount(
+        <Carousel cellAlign="left">
+          <p>Slide 1</p>
+          <p>Slide 2</p>
+          <p>Slide 3</p>
+        </Carousel>
+      );
+      const instance = wrapper.instance();
+      const spy = jest.spyOn(instance, 'onResize');
+      instance.onResize();
```

Comment:
Same comment as https://github.com/FormidableLabs/nuka-carousel/pull/328#discussion_r181909611

You need to actually test the new code you added (aka the `props.onResize` thing)